src/main/java/org/opensearch/ad/task/ADBatchTaskRunner.java

```diff
@@ -30,7 +30,7 @@
 import static org.opensearch.ad.settings.AnomalyDetectorSettings.MAX_TOP_ENTITIES_LIMIT_FOR_HISTORICAL_ANALYSIS;
 import static org.opensearch.ad.settings.AnomalyDetectorSettings.NUM_MIN_SAMPLES;
 import static org.opensearch.ad.stats.InternalStatNames.JVM_HEAP_USAGE;
-import static org.opensearch.ad.util.ParseUtils.isNullOrEmpty;
+import static org.opensearch.timeseries.model.ParseUtils.isNullOrEmpty;
```

Comment:
isNullOrEmpty is standard though -- is the util class being deprecated? Or is there a nuance in the definitions?

Reply:
isNullOrEmpty is commonly used to check if a string is null or empty. 

Good catch. The util class is not deprecated. Will move ParseUtils to util package.